pootle/apps/virtualfolder/utils.py

```diff
@@ -217,3 +218,39 @@ def update_stores(self):
         for store in removed:
             if store.priority == self.vf.priority:
                 store.set_priority()
+
+
+class DirectoryVirtualFoldersDataTool(RelatedStoresDataTool):
+    group_by = (
+        "store__vfolders__title", )
+
+    max_fields = RelatedStoresDataTool.max_fields + (
+        "store__vfolders__priority", "store__vfolders__is_public")
+
+    @property
+    def all_stat_data(self):
+        return self.data_model.filter(
+            store__pootle_path__startswith=self.context.pootle_path)
+
+    def vfolder_is_visible(self, vfolder):
+        return (
+            vfolder["store__vfolders__is_public"]
+            and (vfolder["critical"]
+                 or vfolder["suggestions"]
+                 or (vfolder["store__vfolders__priority"] > 1
```

Comment:
Use `>= 1` as in https://github.com/translate/pootle/blob/master/pootle/apps/virtualfolder/models.py#L161